src/test/scripts/installDependencies.R

```diff
@@ -35,4 +35,5 @@ custom_install("outliers");
 custom_install("caret");
 custom_install("Sigmoid");
 custom_install("DescTools");
+custom_install("mice");
```

Comment:
Why is this required?

Reply:
Thank you for your comments. I have fixed the issue as per your suggestions. I have also tried to fix variable names.